presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -403,6 +403,72 @@ public static Slice upper(@SqlType("varchar(x)") Slice slice)
         return toUpperCase(slice);
     }
 
+    private static Slice pad(Slice text, long inputResultSize, Slice fillCharacters, int paddingOffset)
```

Comment:
Name the parameter `padString` rather than `fillCharacters` and make the same change everywhere.